### DISCUSSION TITLE
an API endpoint to update charging price at specific geofence

### DISCUSSION THREAD

**Discussion by OrangeBean** [teslamate-org/teslamate#3573] [General]
Hello folks,

I'd like to have an API end point or some alternative way to update price at the specific geo-fence via an integration.

Here is the usecase. I have electricity provider who publish electricity supply cost every 30min (might be more frequently). I'd like update charging cost at home location based on the provided price to get accurate cost per charge.

Not sure if API exists or where to find doco about TeslaMate API. I'm not an Elixer developer and couldn't navigate it.

**Comment by vincep5** [comment]
I'd like this too.  I manually go back to update each and every charge for the billing cycle after knowing how much my Real Time pricing is for the cycle.  If there was a way to manually run it via command prompt (like we can do for [ending charges and drives](https://docs.teslamate.org/docs/maintenance/manually_fixing_data)), I could automate or have it done quicker.

**Comment by dborup** [comment]
Was searching for this possibility and ended here, any chance of this support in the future?

**Comment by Gerben321** [comment]
Same here! I'm building something that collects charge costs from the charge providers that I use. I can update the Postgres database, but that's really not preferred obviously.
